charts/openshift-metering/templates/presto/presto-coordinator-config.yaml

```diff
@@ -13,13 +13,29 @@ data:
     node.environment=production
 
   config.properties: |
-    http-server.http.port=8080
-    jmx.rmiserver.port=8081
-    jmx.rmiregistry.port=8081
     coordinator=true
+    node-scheduler.include-coordinator={{ .Values.presto.spec.presto.config.nodeSchedulerIncludeCoordinator }}
     discovery-server.enabled=true
+
+{{- if .Values.presto.spec.presto.config.useTLS }}
+    http-server.http.enabled=false
+    http-server.https.enabled=true
+    http-server.https.port=8080
+    http-server.https.keystore.path=/opt/presto/tls/keystore.jks
+    http-server.https.keystore.key=password
+    node.internal-address=presto
```

Comment:
I think we want to do this the same way that we do the workers, using the podIP. It shouldn't need to match the discovery URI.

Reply:
Update: we've decided to go with hostnames using statefulsets. So this can use `FQDN` instead.